ui/app/components/http-requests-dropdown.js

```diff
@@ -0,0 +1,49 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+/**
+ * @module HttpRequestsDropdown
+ * HttpRequestsDropdown components are used to render a dropdown that filters the HttpRequestsBarChart.
+ *
+ * @example
+ * ```js
+ * <HttpRequestsDropdown @counters={counters} />
+ * ```
+ *
+ * @param counters=null {Array} - A list of objects containing the total number of HTTP Requests for each month. `counters` should be the response from the `/internal/counters/requests` endpoint which looks like:
+ * COUNTERS = [
+ *    {
+ *       "start_time": "2019-05-01T00:00:00Z",
+ *       "total": 50
+ *     }
+ * ]
+ */
+
+export default Component.extend({
+  classNames: ['http-requests-dropdown'],
+  counters: null,
+  timeWindow: 'All',
+  options: computed('counters', function() {
+    let counters = this.counters || [];
+    let options = [];
+    if (counters.length) {
+      const years = counters.reduce((uniqueYears, counter) => {
```

Comment:
Ember has a bunch of array extensions (I think most are from lodash maybe?) - so you could do this with counters.map(/*return year here*/).uniq()

https://api.emberjs.com/ember/3.10/classes/ArrayProxy/methods/uniq?anchor=uniq